tests/Authentication/TokenFilterTest.php

```diff
@@ -14,9 +15,10 @@
 /**
  * @internal
  */
-final class TokenFilterTest extends CIDatabaseTestCase
+final class TokenFilterTest extends CIUnitTestCase
```

Comment:
Shouldn't this extend the new DatabaseTestCase?

Reply:
Done.